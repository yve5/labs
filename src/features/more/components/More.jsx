import React from 'react';
import { Link } from 'react-router-dom';
import { MORE_LINKS } from '../resources/constants';
import { HOME_URL } from '../../home/resources/constants';

const More = () => (
  <div className="container">
    <Link className="container__nav" to={HOME_URL}>
      Back to home
    </Link>

    <h1>More...</h1>

    <ul>
      {MORE_LINKS.map(({ href, label }, index) => (
        <li key={index}>
          <a href={href} target="blank">
            {label}
          </a>
        </li>
      ))}
    </ul>
  </div>
);

export default More;
