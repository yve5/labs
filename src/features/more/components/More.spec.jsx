import { create } from 'react-test-renderer';
import { HashRouter } from 'react-router-dom';
import More from './More';

describe('More', () => {
  it('should match expected snapshot. default', () => {
    const component = create(
      <HashRouter>
        <More />
      </HashRouter>
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
