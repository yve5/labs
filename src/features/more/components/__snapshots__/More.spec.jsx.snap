// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`More should match expected snapshot. default 1`] = `
<div
  className="container"
>
  <a
    className="container__nav"
    href="#/"
    onClick={[Function]}
  >
    Back to home
  </a>
  <h1>
    More...
  </h1>
  <ul>
    <li>
      <a
        href="https://yve5.github.io/visio"
        target="blank"
      >
        Room booking
      </a>
    </li>
    <li>
      <a
        href="https://yve5.github.io/showcase"
        target="blank"
      >
        Showcase
      </a>
    </li>
    <li>
      <a
        href="https://yve5.github.io/contract"
        target="blank"
      >
        Contracts
      </a>
    </li>
    <li>
      <a
        href="https://yve5.github.io/tiles"
        target="blank"
      >
        Tiles
      </a>
    </li>
  </ul>
</div>
`;
